source/_patterns/02-organisms/components/hero-banner~article-poa.json

```diff
@@ -0,0 +1,37 @@
+{
+  "image": {
+    "fallback": {
+      "defaultPath": "https://unsplash.it/1114/336",
+      "altText": ""
+    }
+  },
+  "subjects": {
+    "list": [
+      {
+        "name": "Biochemistry",
+        "url": "#"
+      },
+      {
+        "name": "Biophysics and Structural Biology",
+        "url": "#"
+      }
+    ]
+  },
+  "title": "Molecular basis for multimerization in the activation of the epidermal growth factor",
+  "titleLength": "medium",
+  "url": "#",
+  "summary": "The epidermal growth factor receptor (EGFR) is activated by dimerization, but activation also generates higher-order multimers, whose nature and function are poorly understood. We have characterized ligand-induced dimerization and multimerization of EGFR using single-molecule analysis, and show that multimerization can be blocked by mutations in a specific region of Domain IV of the extracellular module. These mutations reduce autophosphorylation of the C-terminal tail of EGFR and attenuate phosphorylation of phosphatidyl inositol 3-kinase, which is recruited by EGFR. The catalytic activity of EGFR is switched on through allosteric activation of one kinase domain by another, and we show that if this is restricted to dimers, then sites in the tail that are proximal to the kinase domain are phosphorylated in only one subunit. We propose a structural model for EGFR multimerization through self-association of ligand-bound dimers, in which the majority of kinase domains are activated cooperatively, thereby boosting tail phosphorylation.",
+  "authors": "Yongjian Huang et al.",
```

Comment:
Update authors string so that it uses the ellipsis format.

Reply:
The authors are updated without "et al." string